packages/models-library/src/models_library/api_schemas_dynamic_sidecar/containers.py

```diff
@@ -0,0 +1,19 @@
+from pydantic import BaseModel, NonNegativeFloat, validator
+
+
+class InactivityResponse(BaseModel):
+    is_inactive: bool
+    seconds_inactive: NonNegativeFloat | None = None
+
+    @validator("seconds_inactive", always=True)
+    @classmethod
+    def ensure_seconds_inactive_is_set_correctly(cls, v, values):
```

Comment:
Just wondering here...
What is the point of having both a boolean and an optional value?
basically... if seconds_inactive is None, then it is active, if it is a Positive value it means it is inactive since that time. no need for is_inactive here...
and what means inactivity of 0 seconds?

Reply:
Think I see the point here.
- if `seconds_inactive` is set, then it is for sure active
- if `seconds_inactive` is `None`, then it is for sure inactive

I can make transform `is_ianctive` into a property.

Reply:
agree